modules/sagemaker/sagemaker-model-monitoring/sagemaker_model_monitoring/utils.py

```diff
@@ -0,0 +1,17 @@
+from hashlib import md5
+from typing import Any
+
+
+def generate_unique_id(*args: Any) -> str:
+    """
+    Generate a shortened hex digest from a list of arguments.
+
+    Any non-string arguments are cast to a string.
+    """
+    hash = md5()
+    for arg in args:
+        if not isinstance(arg, str):
+            arg = str(arg)
+        hash.update(arg.encode())
+        hash.update(b"\x00")
+    return hash.hexdigest()[:10]
```

Comment:
nit - I think you can just cast all to string and would be able to re-write this into a one-liner list comprehension